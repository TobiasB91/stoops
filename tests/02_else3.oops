CLASS Main IS
    METHOD main IS
        a : Integer;
    BEGIN
        READ a;
        IF a = 1 THEN
            WRITE 65;
        ELSEIF a = 2 THEN
            WRITE 66;
        ELSE
            WRITE 67;
        ELSEIF a = 3 THEN | Fehler
            WRITE 68;
        END IF
    END METHOD
END CLASS
