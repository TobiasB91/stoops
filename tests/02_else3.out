Fehler in Zeile 12, Spalte 9: Unerwartetes Symbol ELSEIF
